tests/test_estimators.py

```diff
@@ -88,6 +92,14 @@ def test_estimators_support_dataframe_indexes(estimator):
     assert_array_equal(idx.ravel(), moscow.index)
 
 
+@pytest.mark.parametrize("estimator", TEST_ESTIMATORS)
+def test_estimators_support_lists(estimator):
+    """All estimators should fit and predict data stored as lists."""
+    X, y = load_moscow_stjoes(return_X_y=True)
+    estimator = estimator().fit(X.tolist(), y.tolist())
+    estimator.predict(X.tolist())
```

Comment:
This is nice assurance as well.  I assume this is where you found the `CCA` bug.

Reply:
Exactly!